src/game.rs

```diff
@@ -315,13 +315,15 @@ fn process_game(
         };
     }
 
-    if let Some(Action::MainMenu)
-    | Some(Action::Help)
-    | Some(Action::UseFood)
-    | Some(Action::UseDose)
-    | Some(Action::UseCardinalDose)
-    | Some(Action::UseDiagonalDose)
-    | Some(Action::UseStrongDose) = option
+    if let Some(
+        Action::MainMenu
+        | Action::Help
+        | Action::UseFood
+        | Action::UseDose
+        | Action::UseCardinalDose
+        | Action::UseDiagonalDose
+        | Action::UseStrongDose,
+    ) = option
```

Comment:
I've actually done this but the Rust version in the CI was older and didn't support it. Happy to merge it now that it does.